mongodbforms/documents.py

```diff
@@ -280,13 +280,13 @@ def __init__(self, options=None):
         self.embedded_field = getattr(options, 'embedded_field_name', None)
         self.formfield_generator = getattr(options, 'formfield_generator',
                                            _fieldgenerator)
-        
+
         self._dont_save = []
-        
-        self.labels = False
-        self.help_texts = False
 
-        
+        self.labels = getattr(options, 'labels', None)
+        self.help_texts = getattr(options, 'help_texts', None)
```

Comment:
I believe this is for Django 1.8. It's not a new commit though:

https://github.com/Work4Labs/django-mongodbforms/commit/5438a1dc206121e207d279cf0e5e3b3e55157d4a